### ISSUE TITLE
[Req] turktorrent

### ISSUE THREAD

**Issue by fevkalbeser** [Jackett/Jackett#4268]
Address: http://turktorrent.us/

Turkish torrent website for Movies, TV Series, Games etc. Invite needed. I can provide one.
Thanks.

**Comment by kaso17** [comment]
If you know a little HTML/CSS you can try to add it yourself.
See https://github.com/Jackett/Jackett/wiki/Definition-format for details.
If you've any questions feel free to ask.

If the tracker already provides some kind of API (e.g. gazelle based trackers) please let us know. Then the Cardigann definition approach might be not not the best solution

**Comment by kekec777** [comment]
Any news / updates on this?
I was gonna req. the same site, but saw this first.

**Comment by garfield69** [comment]
looks like you need to pay for a login?

**Comment by kekec777** [comment]
Hmm i think you need a invitation 
I could send one? :)

**Comment by garfield69** [comment]
If you send an invite to garfield69 at outlook.com I will have a go building a cardigann  indexer. 
But if the site turns out to require a native c# indexer then I will not be able to complete the task.

**Comment by kekec777** [comment]
I have sent an invite to #garfield69

**Comment by garfield69** [comment]
thanks, I've registered successfully.

**Comment by garfield69** [comment]
https://turktorrent.us/ cloudflare Web server is returning an unknown error Error code 520

**Comment by garfield69** [comment]
online

**Comment by kekec777** [comment]
Hmm it seems they have implemented hcaptcha 
Any idear how we can get pass this?

**Comment by garfield69** [comment]
@kekec777  the turktorrent indexer config is already using cookie+useragent. 
Login onto the site with a browser on the same machine that is running Jackett and copy/paste the cookie and UA to the turktorrent indexer config. 
This is working for me, at least until the session cookie expires again. There is no automated solution available.

**Comment by kekec777** [comment]
> @kekec777 the turktorrent indexer config is already using cookie+useragent. Login onto the site with a browser on the same machine that is running Jackett and copy/paste the cookie and UA to the turktorrent indexer config. This is working for me, at least until the session cookie expires again. There is no automated solution available.

Hmm im getting this error:
[turktorrent] (updating) Login failed: Selector "div#member_info_bar" didn't match

And for you information - i have not used flaresolverr before so i just installed it in a docker and connected jackett with it.
The flaresolverr log shows :

Challenge detected. Title found: Just a moment...
Challenge solved!

EDIT:
Its not everytime it solves the challenge 
It fails with:
Error: Error solving the challenge. Message: invalid session id

**Comment by garfield69** [comment]
yes, they have enabled the cloudflare `please wait while we check the connection is secure` protection, which the flaresolverr service can solve.
however, the login page that follows is still the original one with the hcaptcha built in, which cannot be solved by flaresolverr so ATM the cookie method is still the only way for jackett to access the site.

**Comment by garfield69** [comment]
while turktorrent have this heightened level of anti-DDoS protection, Jackett is effectively blocked.
Even with a session cookie from the browser, the cloudflare challenge that kicks in at every session open, changes the cf-clearance cookie, effectively rendering the browsers session cookie useless.

**Comment by kekec777** [comment]
> while turktorrent have this heightened level of anti-DDoS protection, Jackett is effectively blocked. Even with a session cookie from the browser, the cloudflare challenge that kicks in at every session open, changes the cf-clearance cookie, effectively rendering the browsers session cookie useless.

So in other words , we cant use turktorrent in jackett?

**Comment by ilike2burnthing** [comment]
Until that additional protection is dropped.

**Comment by garfield69** [comment]
- https://turktorrent.us/ > We are old and tired... 2006-2025
- I'm going...  2006-2025

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://turktorrent.us/ cloudflare connection closed or timeout

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://turktorrent.us/?p=torrents&pid=32&search_type=name&sortOptions[sortBy]=added&sortOptions[sortOrder]=desc > 502 Bad Gateway

Had to change the login test today with 7d5c48d873507874bbccec38950afc3d1dbbc690

But in addition something has changed with the torrent page processing because I cannot tell why the GET generated by the indexer is not working (returns 502, as does using that full path in the browser), other than the path appears to be just https://turktorrent.us/?p=torrents&pid=32, but its not a POST, so I don't know where the parms are coming from, so maybe there is JS involved?

**Comment by ilike2burnthing** [comment]
They added a couple of S's, who knows why 🙄 

No idea if this is new, but when I try to view a poster, I get a FlareSolverr challenge.

More importantly though (although again, no idea if it's new), downloads are now locked to VIPs. You seeing the same thing? If so, wanna add `PAY2DL` to the description?

**Comment by garfield69** [comment]
I'll drop the posters and add a pay2dl tag.

**Comment by garfield69** [comment]
https://turktorrent.us/?p=torrents&pid=32 > timeout

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/4268#issuecomment-4184063814

**Comment by ilike2burnthing** [comment]
back up
